doc/source/dev/contributor/quickstart_ubuntu.rst

```diff
@@ -14,7 +14,8 @@ This quickstart guide will cover:
 * performing an in-place build of SciPy; and
 * creating a virtual environment that adds this development version of SciPy to the Python path
 
-in Ubuntu.
+in Ubuntu 16.04. *Users running Windows can follow these
+instructions after setting up* `Windows Subsystem for Linux`_ *or an Amazon EC2 instance with Ubuntu 16.04 LTS.*
```

Comment:
perhaps slightly orthogonal to the discussion, but not completely irrelevant since I've done some releases using WSL--I can't actually tell if the newer "more native" versions of WSL allow:

1. Use of Docker on Ubuntu on Windows -- IIRC this is surprisingly complex because the low-level things are still slightly different from pure Linux.
2. Use of apps that open X11-style windows on Ubuntu on Windows

The version I'm using is probably one of the earlier ones, where neither of those is possible.
Or more accurately, you have to look for blog posts and go through involved work-arounds rather than having native support for that. Maybe not true anymore though!

Reply:
I can't speak to #1. For #2, I was able to do [this](https://www.howtogeek.com/261575/how-to-run-graphical-linux-desktop-applications-from-windows-10s-bash-shell/). Unfortunately, I couldn't use Spyder via X11 forwarding. (In case anyone is interested - an alternative to that is to run Spyder on Windows and have it execute code on a kernel running on WSL. I've had a little trouble editing files on WSL in the Spyder running on Windows, but it sounds like it's [possible, only the run button won't work](https://github.com/spyder-ide/spyder/issues/3065).)

Reply:
@tylerjereddy regarding (2) I've been using visual studio code with the [Remote - WSL](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) plugin and it's worked well for me so far.

Reply:
Ooh that sounds like exactly what I want Spyder to do. I will try this in the meantime.

Reply:
Wow, this is great. Finally, a relatively simple and complete way of doing SciPy development from Windows.

Reply:
This is cool stuff--I'll mark as resolved though so we don't get too side-tracked on this specific PR.